src/xai_grok/schemas/message_tool_choice.py

```diff
@@ -0,0 +1,8 @@
+import typing
+
+import pydantic
+
+
+class MessageToolChoice(pydantic.BaseModel):
+    name: typing.Optional[str] = None
+    type: str
```

Comment:
The `type` field is required but lacks a description of its purpose and valid values. This could lead to incorrect usage. Consider adding a docstring or field description to clarify what `type` represents.